git-commit/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "git-commit"
+version = "0.1.0"
+license = "MIT OR Apache-2.0"
+edition = "2021"
+authors = [
+  "Alexis Sellier <alexis@radicle.xyz>",
+  "Fintan Halpenny <fintan.halpenny@gmail.com",
```

```suggestion
  "Fintan Halpenny <fintan.halpenny@gmail.com>",
```
If you copy pasted this, I'd check the other tomls just in case xD

---

Ahahaha, no I'm pretty sure I typed it :see_no_evil: